attributes/default.rb

```diff
@@ -18,5 +18,5 @@
 # limitations under the License.
 #
 
-default['gluster']['version'] = '3.4'
+default['gluster']['version'] = '3.6'
```

Comment:
Sounds sensible to me. I didn't realise 3.7 was out of beta... it's been a while since I looked!